src/test/lua/exasolrls/test_adapter_properties.lua

```diff
@@ -0,0 +1,51 @@
+local luaunit = require("luaunit")
+local props = require("exasolrls.adapter_properties")
+
+test_rls_adapter = {}
+
+function test_rls_adapter.test_validate_properties()
+    local tests = {
+        {
+            properties = {},
+            expected = 'Missing mandatory property "SCHEMA_NAME"',
+        },
+        { 
+            properties = {SCHEMA_NAME = ""},
+            expected = 'Missing mandatory property "SCHEMA_NAME"',
+        },
+        {
+            properties = {SCHEMA_NAME = "THE_SCHEMA", TABLE_FILTER = ""},
+            expected = "Table filter property must not be empty."
```

Comment:
Expected error messages should also contain the error code.

Reply:
I will change that with #88 in a refactoring. It should not be a blocker for the 1.0.0 release though.